src/components/CallToAction/ExtendedGetInstanceCallToAction.js

```diff
@@ -54,23 +54,22 @@ const submitToNetlifyForms = async ({
       body: formData,
     });
 
-    trackRequestTrial({
-      name,
-      email,
-      locationSearch,
-    });
-    trackPlausibleEvent(netlifyFormName, {
-      name,
-      email,
-      locationSearch,
-    });
-
-    if (subToNewsletter) {
-      trackSubscribe({
-        name,
+    if (resp.ok) {
+      trackRequestTrial({
```

Comment:
Bots likely submit nonsense all the time and cause failed requests. We shouldn't count these for analytics.